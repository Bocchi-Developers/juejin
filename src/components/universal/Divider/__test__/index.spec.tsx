import { describe, expect, it } from 'vitest'

import { render } from '@testing-library/react'

import mountTest from '~/test/mountTest'

import { Divider } from '..'

mountTest(Divider)

describe('Card', () => {
  it('snapshot', () => {
    const markdown = render(<Divider className={'test'} />)
    expect(markdown.container).matchSnapshot()
  })
})
