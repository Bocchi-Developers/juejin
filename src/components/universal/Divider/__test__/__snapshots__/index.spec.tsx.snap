// Vitest Snapshot v1

exports[`Card > snapshot 1`] = `
<div>
  <div
    class="arco-divider arco-divider-horizontal test"
    role="separator"
    style="margin: .6em 0px 0px 0px;"
  />
</div>
`;
